### ISSUE TITLE
Extract to zombie file

### ISSUE THREAD

**Issue by xlecours** [aces/cbrain#781]
Extracting files from a FileCollection sitting on a DataProvider which CBRAIN do not have the write permission creates new file records that can't be deleted.

Error message when deleting: 
```
Error when deleting/unregistering file(s)
[<datetime>] For 1 file, Error: provider <dataprovidername> is read_only.:
<filename>
```

**Comment by prioux** [comment]
Good one, will handle right away.

**Comment by prioux** [comment]
Fixed by #812
